lib/game_team.rb

```diff
@@ -45,16 +36,12 @@ def self.worst_coach(season_id)
     worst = wins_by_coach.min_by {|coach, percent| percent}
     worst[0]
   end
-
+#what does this do?
   def self.game_team_shots_goals_count(arr_games)
```

Comment:
I actually dont know what this method does. its not tied to any other method, it may be a relic from the past. I should get rid of that comment though